<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1002</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>171</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="fileSelectButton">
       <property name="text">
        <string>Open File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="fileNameLabel">
       <property name="text">
        <string>Current: </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>531</width>
      <height>481</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>День</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Месяц</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Год</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Разница с i+1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Разница с сегодня</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Завтра</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>40</y>
      <width>221</width>
      <height>182</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="prevButton">
       <property name="text">
        <string>Предыдущий день</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="isLeapButton">
       <property name="text">
        <string>Год високосный?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="weekButton">
       <property name="text">
        <string>Номер недели</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bdayButton">
       <property name="text">
        <string>Дней до дня рождения</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QSpinBox" name="spinDay">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>320</y>
      <width>51</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinMonth">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>320</y>
      <width>51</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinYear">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>320</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>270</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;День рождения:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="todayLabel">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>410</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>370</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Сегодня:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="editButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>280</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonData2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ввод новой даты</string>
    </property>
   </widget>
   <widget class="QLabel" name="newdayLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Новая дата:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1002</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDates">
    <property name="title">
     <string>Dates</string>
    </property>
   </widget>
   <addaction name="menuDates"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
